.vscode/cspell.json

```diff
@@ -1,15 +1,119 @@
 {
-  "version": "0.1",
+  "version": "0.2",
   "language": "en",
   "languageId": "python",
   "dictionaries": [
     "powershell",
-    "python"
+    "python",
+    "softwareTerms",
+    "en_us",
+    "en-gb"
   ],
   "ignorePaths": [
     "**/tests/recordings/**",
-    ".vscode/cspell.json"
+    ".vscode/cspell.json",
+    "*.parquet",
+    "*.cspkg",
+    "*.avro",
+    "*.tgz",
+    "*.pfx",
+    "sdk/**/azure-mgmt**/**",
+    "sdk/**/azure-cognitiveservices**",
+    "sdk/**/_generated/**",
+    "dev_requirement.txt"
```

Comment:
Should this be **/**requirements.txt?

Reply:
I see `dev_requirement.txt`, `dev_requirements.txt`, `requirement.txt`, and `requirements.txt` -- For that, `**/*requirement*.txt` makes sense.